### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Galileo-Galilei**
## Description

- Add Nok & Joel as contributors 
- Add link to tutorial in the README before release to make it more discoverable
- Update doc index link to fix RTD builds which is empty

## Checklist

- [X] Read the [contributing](../CONTRIBUTING.md) guidelines
- [ ] Open this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Update the documentation to reflect the code changes
- [ ] Add a description of this change and add your name to the list of supporting contributions in the [`CHANGELOG.md`](../CHANGELOG.md) file. Please respect [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines.
- [ ] Add tests to cover your changes

## Notice

- [X] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

**Comment by Galileo-Galilei** [comment]
@datajoely @noklam Are you fine if I add your names to the README?

**Review by noklam — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ pip install --upgrade git+https://github.com/Galileo-Galilei/kedro-pandera.git
 
 The documentation contains:
 
-- [A  "hello world" example]()
+- [A  "hello world" example](https://kedro-pandera.readthedocs.io/en/stable/source/01_tutorials/01_getting_started.html)
```
Comment (noklam) [inline README.md:41]:
```suggestion
- [A "hello world" example](https://kedro-pandera.readthedocs.io/en/stable/source/01_tutorials/01_getting_started.html)
```

**Comment by noklam** [comment]
https://github.com/kedro-org/kedro-starters/issues/143
OmegaConfigLoader is going to be default before 0.19

**Comment by noklam** [comment]
I just go ahead and merge it as I saw Joel thumbs up.

**Pull request merged**